src/components/transactionsV2/filters/messageFieldGroup.js

```diff
@@ -60,11 +84,21 @@ class MessageFieldGroup extends React.Component {
             placeholder={t('Write message')}
             maxLength={100}
             onKeyDown={handleKeyPress}
-            className={`${styles.input} ${fields.message.error ? 'error' : ''}`} />
+            className={`${styles.input} ${fields.message.error ? 'error' : ''}`}
+          />
+          <CircularProgress max={62} value={byteCount}
+            className={styles.byteCounter} />
+          <img
+            className={`${styles.status} ${!fields.message.loading && filters.message ? styles.show : ''}`}
+            src={ fields.message.error ? svg.alert_icon : svg.ok_icon} />
         </div>
-        <span className={`${styles.feedback} ${this.state.feedback ? styles.show : ''}`}>
-          {this.state.feedback}
-        </span>
+        <Feedback
+          className={styles.feedback}
+          show={!!this.state.feedback}
```

Comment:
here you you can use the error value of the value... I dont know if it to much difficult or not